src/pages/home/report/ReportActionCompose.js

```diff
@@ -308,10 +308,10 @@ class ReportActionCompose extends React.Component {
     render() {
         // eslint-disable-next-line no-unused-vars
         const hasMultipleParticipants = lodashGet(this.props.report, 'participants.length') > 1;
+        const hasConciergeParticipant = _.contains(this.props.report.participants, CONST.EMAIL.CONCIERGE);
```

Comment:
Only this line has been changed compared to the previous PR https://github.com/Expensify/Expensify.cash/pull/2740

Before:
```
const hasConciergeParticipant = this.props.report.participants.includes(CONST.EMAIL.CONCIERGE);
```